include/boost/geometry/strategies/spherical/closest_points_cross_track.hpp

```diff
@@ -0,0 +1,450 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2019, Oracle and/or its affiliates.
+
+// Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_SPHERICAL_CLOSEST_POINTS_CROSS_TRACK_HPP
+#define BOOST_GEOMETRY_STRATEGIES_SPHERICAL_CLOSEST_POINTS_CROSS_TRACK_HPP
+
+#include <boost/geometry/algorithms/detail/closest_points/result.hpp>
+
+#include <boost/geometry/strategies/closest_points.hpp>
+#include <boost/geometry/strategies/spherical/distance_cross_track.hpp>
+
+#include <boost/geometry/formulas/spherical.hpp>
+
+#include <iostream>
+
+namespace boost { namespace geometry
+{
+
+namespace strategy { namespace closest_points
+{
+
+template
+<
+        typename CalculationType = void,
+        typename Strategy = distance::haversine<double, CalculationType>
+>
+class cross_track
```

Comment:
So this could be `spherical_point_segment` right? Plus, Strategy is probably not needed as an argument.

Reply:
I named it like this to be consistent with distance. I prefer to changed them both at once in a new PR.

Reply:
`Strategy` is needed to define to comparable strategy used to compute the comparable distance.